agent/packaging/ubuntu_xenial/kontena-agent/etc/kontena-agent.env

```diff
@@ -0,0 +1,11 @@
+# Set URL of your kontena server
+#KONTENA_URI=wss://localhost:8443
+
+# Set kontena token
+#KONTENA_TOKEN=
+
+# kontena peer interface (overlay network interface)
+KONTENA_PEER_INTERFACE=eth1
+
+# kontena version
+KONTENA_VERSION=latest
```

Comment:
That's a good question. I was thinking that using latest would enable smooth upgrade with just `systemctl restart kontena-agent` as it works on CoreOS that are provisioned with our cli plugins. @jakolehm What do you think, should we pin the version to the package version?